dkc/core/rest/file.py

```diff
@@ -64,7 +65,8 @@ def _validate_unique_folder_siblings(self, attrs):
 
 class FileUpdateSerializer(FileSerializer):
     class Meta(FileSerializer.Meta):
-        read_only_fields = FileSerializer.Meta.read_only_fields + ['folder']
+        fields = FileSerializer.Meta.fields + ['blob']
+        read_only_fields = FileSerializer.Meta.read_only_fields + ['folder', 'size']
```

Comment:
Right now, there's nothing that prevents subsequent PATCH requests from continuing to overwrite the `blob` again and again. Is this something that will be addressed in the future?

Reply:
Yes, https://github.com/girder/dkc-next/issues/139 captures this and other policies we want to enforce.